File: csharp/MapleStory.OpenAPI/Src/Dto/Character/CharacterCashItemEquipmentColoringPrismDTO.cs
using Newtonsoft.Json;

namespace MapleStory.OpenAPI.Dto
{
    /// <summary>
    /// 캐릭터 캐시 장비 컬러링프리즘
    /// </summary>
    public class CharacterCashItemEquipmentColoringPrismDTO
    {
        /// <summary>
        /// 컬러링프리즘 색상 범위
        /// </summary>
        [JsonProperty("color_range")]
        public string ColorRange { get; set; }

        /// <summary>
        /// 컬러링프리즘 색조
        /// </summary>
        [JsonProperty("hue")]
        public long Hue { get; set; }

        /// <summary>
        /// 컬러링프리즘 채도
        /// </summary>
        [JsonProperty("saturation")]
        public long Saturation { get; set; }

        /// <summary>
        /// 컬러링프리즘 명도
        /// </summary>
        [JsonProperty("value")]
        public long Value { get; set; }
    }
}

File: java/src/main/java/dev/spiralmoon/maplestory/api/tms/dto/character/CharacterCashItemEquipmentColoringPrismDTO.java
package dev.spiralmoon.maplestory.api.tms.dto.character;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

/**
 * 現金道具彩色稜鏡資訊
 */
@AllArgsConstructor
@NoArgsConstructor
@Data
@ToString
public class CharacterCashItemEquipmentColoringPrismDTO implements dev.spiralmoon.maplestory.api.common.dto.character.CharacterCashItemEquipmentColoringPrismDTO {

    /**
     * 彩色稜鏡顏色範圍
     */
    @JsonProperty("color_range")
    private String colorRange;

    /**
     * 彩色稜鏡色調
     */
    @JsonProperty("hue")
    private long hue;

    /**
     * 彩色稜鏡彩度
     */
    @JsonProperty("saturation")
    private long saturation;

    /**
     * 彩色稜鏡明度
     */
    @JsonProperty("value")
    private long value;
}
